runtime/src/lib.rs

```diff
@@ -504,28 +519,28 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl encointer_communities_rpc_runtime_api::CommunitiesApi<Block> for Runtime {
-		fn get_cids() -> Vec<CommunityIdentifier> {
-			EncointerCommunities::get_cids()
-		}
+	#[cfg(feature = "runtime-benchmarks")]
+	impl frame_benchmarking::Benchmark<Block> for Runtime {
+		fn benchmark_metadata(extra: bool) -> (
+			Vec<frame_benchmarking::BenchmarkList>,
+			Vec<frame_support::traits::StorageInfo>,
+		) {
+			use frame_benchmarking::{list_benchmark, Benchmarking, BenchmarkList};
+			use frame_support::traits::StorageInfoTrait;
+			use frame_system_benchmarking::Pallet as SystemBench;
 
-		fn get_name(cid: &CommunityIdentifier) -> Option<PalletString> {
-			EncointerCommunities::get_name(cid)
-		}
-	}
+			let mut list = Vec::<BenchmarkList>::new();
 
-	impl encointer_bazaar_rpc_runtime_api::BazaarApi<Block, AccountId> for Runtime {
```

Comment:
@brenzi : Regarding your compiler error. This is because you remove the runtime api implementation here.

Reply:
aarghh. an unmindful copy-paste from upstream